en/docs/chapter_tree/binary_tree_traversal.md

```diff
@@ -1,14 +1,14 @@
 # Binary tree traversal
 
-From the perspective of physical structure, a tree is a data structure based on linked lists, hence its traversal method involves accessing nodes one by one through pointers. However, a tree is a non-linear data structure, which makes traversing a tree more complex than traversing a linked list, requiring the assistance of search algorithms to achieve.
+From a physical structure perspective, trees are data structures based on linked lists. Hence, their traversal method involves accessing nodes one by one through pointers. However, trees are non-linear data structures, which makes traversing a tree more complex than traversing a linked list, requiring the assistance of search algorithms.
```

Comment:
'From a physical structure perspective, trees are data structures based on linked lists.' --> 'From a physical structure perspective, a tree is a linked-list-based data structure.'

"Tree" refers as a data structure, using the plural form feels incorrect

Reply:
'their' --> 'its'

Reply:
This paragraph is discussing one specific data structure—a tree. It is not referring to multiple trees or instances of trees, so maybe the singular form should be sufficient.

Reply:
Thanks, updated.